src/components/panels/OffersPanel/OffersPanel.tsx

```diff
@@ -17,7 +17,7 @@ type tableData = {
 export default function OffersPanel({
   entity: application,
 }: Props): ReactElement {
-  const modelStatusData = useModelStatus();
+  const modelStatusData: any = useModelStatus();
```

Comment:
Aren't we avoiding `any` types?

Reply:
yep, I only added this so that it would compile (it required it with my version of TS for some reason) but I've now updated this to use a custom `TSFixMe` type.